### PR TITLE
LIS/HyperV Platform implemented

### PR DESCRIPTION
**Pull request by papagalu**
LIS/HyperV platform implementation

### CODE REVIEW

File: scripts/lis_hyperv_platform/check_kernel.ps1 [hunk 1]
```diff
@@ -0,0 +1,42 @@
+param(
+    [String] $InstanceName = "something",
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/check_kernel.ps1]: Instance1

File: scripts/lis_hyperv_platform/check_kernel.ps1 [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+function Main {
+    $backend = [HypervBackend]::new(@("localhost"))
+    $instance = [HypervInstance]::new($backend, $InstanceName, "")
+
+    #
+    # wait until the machine restarted
+    # might get ip before restart
+    # any other solution is welcomed
+    #
+    Start-Sleep 100
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/check_kernel.ps1]: Remove this start sleep, as you do not need it anymore

File: scripts/lis_hyperv_platform/main.ps1 [hunk 3]
```diff
@@ -0,0 +1,17 @@
+param(
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/main.ps1]: put generic paths here

File: scripts/lis_hyperv_platform/main.ps1 [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    [String] $VHDPath = "C:/users/dimi/workspace\framework_scripts\lis-pipeline\scripts\lis_hyperv_platform\ubuntu-cloud.vhdx",
+    [String] $ConfigDrivePath = "C:\Users\dimi\workspace\framework_scripts\lis-pipeline\scripts\lis_hyperv_platform/configdrive",
+    [String] $UserdataPath = "C:\Users\dimi\workspace\framework_scripts\lis-pipeline\scripts\lis_hyperv_platform\install_kernel.sh",
+    [String[]] $KernelURL = @(
+        "http://10.7.1.199/debs/linux-headers-4.13.2_4.13.2-10.00.Custom_amd64.deb",
+        "http://10.7.1.199/debs/linux-image-4.13.2_4.13.2-10.00.Custom_amd64.deb",
+        "http://10.7.1.199/debs/hyperv-daemons_4.13.2_amd64.deb"),
+    [String] $MkIsoFS = "C:\Users\dimi\workspace\utils\mkisofs\mkisofs.exe",
+    [String] $InstanceName = "something",
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/main.ps1]: InstanceName1

File: scripts/lis_hyperv_platform/setup_env.ps1 [hunk 5]
```diff
@@ -0,0 +1,24 @@
+param (
+    [String] $VHDPath = "C:/users/dimi/workspace\framework_scripts\lis-pipeline\scripts\lis_hyperv_platform\ubuntu-cloud.vhdx",
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/setup_env.ps1]: put either generic paths or nothing

File: scripts/lis_hyperv_platform/setup_metadata.ps1 [hunk 6]
```diff
@@ -0,0 +1,60 @@
+param(
+    [String] $ConfigDrivePath = "C:\Users\dimi\workspace\framework_scripts\lis-pipeline\scripts\lis_hyperv_platform/configdrive",
```
Comment (ader1990) [inline scripts/lis_hyperv_platform/setup_metadata.ps1]: ditto

File: scripts/lis_hyperv_platform/main.ps1 (hunk 3 above)
Comment (papagalu) [reply scripts/lis_hyperv_platform/main.ps1]: done

**Pull request merged**